anesthetic/samples.py

```diff
@@ -861,6 +866,26 @@ def stats(self, nsamples=None, beta=None):
         samples.set_label('d_G', r'$d_\mathrm{G}$')
 
         samples.label = self.label
+
+        if norm is not None:
```

Comment:
What if `norm` has some but not all of the columns from samples?

Reply:
Then it will throw an error message saying so. Shouldn't usually be a problem, since I expect `norm` to be a `Samples` instance created by a call to `stats` in the first place, which will automatically compute all relevant columns.